platform/lib/platform_web/live/adminland_live/api_token_live.ex

```diff
@@ -120,10 +117,7 @@ defmodule PlatformWeb.AdminlandLive.APITokenLive do
                 <h3 class="mt-2 text-sm font-medium text-gray-900">No API tokens</h3>
                 <p class="mt-1 text-sm text-gray-500">Get started by creating a new API token.</p>
                 <div class="mt-6">
-                  <%= live_patch("New Token",
-                    class: "button ~urge @high",
-                    to: Routes.adminland_index_path(@socket, :api_new)
-                  ) %>
+                  <.link method = "New Token" class = "button ~urge @high" patch =  {Routes.adminland_index_path(@socket, :api_new)}></.link>
```

Comment:
See above; also note the double space after `patch` (you can fix that by running `mix format`).